docs/usage-cli.md

```diff
@@ -102,7 +102,8 @@ Extra header to include in every request. This option can be used multiple times
 Show usage information.<br>
 
 ### --hookfiles, -f
-Specifies a pattern to match files with before/after hooks for running tests<br>
+Specifies a pattern to match files with before/after hooks for running tests. Files are executed in
```

Comment:
This is actually generated from `options.coffee` (should be solved by #775). Edit `options.coffee` and then run `npm run build`.